examples/resources/stackit_loadbalancer/resource.tf

```diff
@@ -0,0 +1,41 @@
+resource "stackit_loadbalancer" "example" {
+  project_id = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+  name       = "example-load-balancer"
+  target_pools = [
+    {
+      name        = "example-target-pool"
+      target_port = 80
+      targets = [
+        {
+          display_name = "example-target"
+          ip           = openstack_compute_instance_v2.example.network.0.fixed_ip_v4
```

Comment:
maybe you should also add here the configuration of the openstack resources. I saw they are defined in the resource description but they should also be here for clarity

Reply:
But then I would remove them from the resource description, right? Otherwise they would show twice in the documentation

Reply:
Yeah I think its better like that. And in the documentation you can reference the example. That way the example can be used as a standalone file by the users to try it out